### PR TITLE
Fix pytorch

### LINKED ISSUE #753: new pytorch release breaks build
the test `tests/network_components_free_test.py` fails

### PR DESCRIPTION
**Pull request by mganahl**
fFx arithmetic behaviour of `Node`, fix and add tests for them.
Fixes #753

### CODE REVIEW

**Comment by mganahl** [comment]
I changed back travis requirement to torch>=1.4.0. eventually we should support complex dtypes for torch as well, that means using torch>=1.6.0 .

**Comment by mganahl** [comment]
@Thenerdstation this is good to go from my side.

File: tensornetwork/network_components.py [hunk 1]
```diff
@@ -569,61 +569,43 @@ def op_protection(self, other: Union[int, float, "Node"]) -> "Node":
...
-      axis_names = self.axis_names
-    else:
-      axis_names = other.axis_names
+    other_tensor = self.op_protection(other)
+    new_tensor = self.backend.addition(self.tensor, other_tensor)
     return Node(
         tensor=new_tensor,
         name=self.name,
-        axis_names=axis_names,
+        axis_names=None,
```
Comment (Thenerdstation) [inline tensornetwork/network_components.py:581]: Is there a reason you changed this?

File: tensornetwork/tests/tensornetwork_symmetric_test.py [hunk 2]
```diff
@@ -44,30 +44,6 @@ def get_random_symmetric(shape, flows, num_charges, seed=10, dtype=np.float64):
...
-
-
-def get_ones(shape, dtype=np.float64):
-  R = len(shape)
-  charges = [U1Charge(np.random.randint(-5, 5, shape[n])) for n in range(R)]
-  flows = list(np.full(R, fill_value=False, dtype=np.bool))
-  indices = [Index(charges[n], flows[n]) for n in range(R)]
-  return BlockSparseTensor.ones(indices=indices, dtype=dtype)
-
-
```
Comment (Thenerdstation) [inline tensornetwork/tests/tensornetwork_symmetric_test.py:70]: Let's not delete these in this PR.

File: tensornetwork/network_components.py (hunk 1 above)
Comment (mganahl) [reply tensornetwork/network_components.py:581]: I feel using the axis names of self is somewhat arbitrary (why not the ones of other?), and can confuse users. This way seems more natural.

File: tensornetwork/tests/tensornetwork_symmetric_test.py (hunk 2 above)
Comment (mganahl) [reply tensornetwork/tests/tensornetwork_symmetric_test.py:70]: I'll add them back

**Pull request merged**